src/main/java/frc/robot/common/hardware/MotorController.java

```diff
@@ -42,6 +43,7 @@ public void setPID(double[] PID) {
     mP = PID[0];
     mI = PID[1];
     mD = PID[2];
+    mFF = PID[3];
```

Comment:
This will break climb, I think.  It creates a new motorcontroller, but has a PID array of only 3 items.

Reply:
It's just a 0 to add, I think FF support is worth it